app/connectors/connectors/jms/activemq-connectors/activemq-model/src/main/java/io/syndesis/connector/jms/ActiveMQConnectorVerifierExtension.java

```diff
@@ -85,6 +86,14 @@ private void verifyCredentials(ResultBuilder builder, Map<String, Object> parame
                     .parameterKey("username")
                     .parameterKey("password")
                     .build());
+        } finally {
+            if (connection != null) {
+                try {
+                    connection.close();
```

Comment:
Perhaps we could use try-with-resources here

Reply:
Connection is not a Closable.